### ISSUE TITLE
[BUG] from_pytest_fixture doesn't seem to work with parametrized fixtures

### ISSUE THREAD

**Issue by dkbarn** [schemathesis/schemathesis#1121]
When using ```schemathesis.from_pytest_fixture()```, if that fixture is parametrized with ```@pytest.fixture(params=...)```, pytest produces an error like this:
```
tests/integration/test_graphql_schema.py::test_graphql_schema FAILED                                                                                                                           [100%]

============================================================================================== FAILURES ==============================================================================================
________________________________________________________________________________________ test_graphql_schema _________________________________________________________________________________________
'Function' object has no attribute 'callspec'

During handling of the above exception, another exception occurred:
The requested fixture has no parameter defined for test:
    tests/integration/test_graphql_schema.py::test_graphql_schema

Requested fixture 'graphql_schema_fixture' defined in:
tests/integration/test_graphql_schema.py:8

Requested here:
./venv/lib/python3.8/site-packages/schemathesis/lazy.py:189
```

**To Reproduce**
Here's an example of what the type of tests I'm trying to run:
```
import pytest
import schemathesis

API_VERSIONS = ["v1", "v2", "v3"]

@pytest.fixture(params=API_VERSIONS)
def graphql_schema_fixture(app):
    endpoint = f"/{request.param}/graphql"
    return schemathesis.graphql.from_wsgi(endpoint, app)

graphql_schema = schemathesis.from_pytest_fixture("graphql_schema_fixture")

@graphql_schema.parametrize()
def test_graphql_schema(case):
    response = case.call_wsgi()
    case.validate_response(response)
```

**Expected behavior**
When generating a schema object from a parametrized fixture, decorating a test function with that schema object would result in multiple iterations of the test function, for each schema produced by the parametrized fixture.

**Environment (please complete the following information):**
 - OS: Linux
 - Python version: 3.8.5
 - Schemathesis version: 3.6.0
 - Spec version: Open API 3.0.2
 - Pytest version: 6.2.3

**Comment by Stranger6667** [comment]
Hi @dkbarn !

Thank you for a very detailed report, I will take a look at it in the next few days.

**Comment by Stranger6667** [comment]
Fixed in `3.6.1`

**Comment by dkbarn** [comment]
Thanks for quick fix! Confirmed that it's now working as expected in 3.6.1.
